plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -241,6 +252,44 @@ func (c *nodePlugin) admitPodEviction(nodeName string, a admission.Attributes) e
 	}
 }
 
+func (c *nodePlugin) admitPVCStatus(nodeName string, a admission.Attributes) error {
+	switch a.GetOperation() {
+	case admission.Update:
+		if !utilfeature.DefaultFeatureGate.Enabled(features.ExpandPersistentVolumes) {
+			return admission.NewForbidden(a, fmt.Errorf("node %q may not update persistentvolumeclaim metadata", nodeName))
+		}
+
+		oldPVC, ok := a.GetOldObject().(*api.PersistentVolumeClaim)
+		if !ok {
+			return admission.NewForbidden(a, fmt.Errorf("unexpected type %T", a.GetOldObject()))
+		}
+
+		newPVC, ok := a.GetObject().(*api.PersistentVolumeClaim)
+		if !ok {
+			return admission.NewForbidden(a, fmt.Errorf("unexpected type %T", a.GetObject()))
+		}
+
+		// make copies for comparison
```

Comment:
can you update this to where the POC ended up (whitelisting specific status fields for nodes to update) - https://github.com/kubernetes/kubernetes/pull/51202/commits/daf8bcef691a5495b1b10192e4f8ff8ff13b08fb#diff-cbf05e22a4f412de387669881892d46bR273

Reply:
sure.

Reply:
fixed